src/api/egenansatt.ts

```diff
@@ -0,0 +1,19 @@
+import { apiBaseUri } from './config';
+import { Egenansatt } from '../models/egenansatt';
+
+export function getEgenAnsatt(fødeselsnummer: string): Promise<Egenansatt> {
+    const uri =
+        `${apiBaseUri}/egenansatt/${fødeselsnummer}`;
+    return fetch(uri, {credentials: 'include'})
+        .then((response) => {
+            if (response.ok) {
+                return response.json();
+            } else if (response.status === 404) {
```

Comment:
Her tolker du 404 som at det gikk bra, bare at det ikke var noe response. Det gir mening for noen endepunkt, som for eksemlel nav-kontor, men tror kanskje ikke det er gir mening her. Bedre å kaste feil, kanskje?

Reply:
kanskje bare fjerne hele else if blokken. Trenger ikke spesialhåndtering på 404